apps/meteor/ee/app/livechat-enterprise/server/api/priorities.ts

```diff
@@ -25,12 +28,46 @@ API.v1.addRoute(
 );
 
 API.v1.addRoute(
-	'livechat/priorities/:priorityId',
+	'livechat/priority',
+	{
+		authRequired: true,
+		permissionsRequired: { POST: { permissions: ['manage-livechat-priorities'], operation: 'hasAny' } },
+	},
+	{
+		async post() {
+			const { name, level } = this.bodyParams;
+			check(name, String);
+			check(level, String);
+			const insert = await createPriority({ name, level });
+			if (insert !== false) {
```

Comment:
insert !== false => if (insert) 🤔

Reply:
yeah, this appears unnecessary with the other changes